js/subscribe.js

```diff
@@ -0,0 +1,44 @@
+const modal = document.getElementById("submitModal");
+const submitSubscription = document.getElementById("submitSubscription");
+const congratsText = document.getElementById("congrats");
+const emailInput = document.getElementById("emailAddress");
+
+// function subscriptionButton() {
+//     modal.style.display = "block";
+// }
+
+const modalOptYes = document.getElementById("modalYes");
+const modalOptNo = document.getElementById("modalNo");
+
+
+
+const validateEmail = () => {
+    console.log(emailInput.value)
+    const regx = /^([a-zA-Z0-9\._]+)@([a-zA-Z0-9])+.([a-z]+)(.[a-z]+)?$/
+
+    console.log(regx.test(emailInput.value));
+    if (regx.test(emailInput.value)) {
+        submitSubscription.disabled = false;
```

Comment:
is the button disabled initially? if not please disable the button initially and only set it to false once the test passes. a nice touch would be to change the button color to a lighter color scheme to indicate disabled state